lib.sh

```diff
@@ -1080,6 +1080,31 @@ case "${1}" in
 esac
 }
 
+# Example:
+# notify_user_gui \
+# "Subject" \
+# "Message"
+#
+# occ_command notification:generate -l "$2" "$admin" "$1"
+notify_user_gui() {
+USER=$(occ_command user:list | awk '{print $2}' | cut -d ":" -f1;)
```

Comment:
`cut -d ":" -f1;`
what  is tis addition doing?

Reply:
run `"occ_command user:list | awk '{print $2}"`  and notice that it adds a `:`, the `cut` command deletes the `:`